plugins/pulp_rpm/plugins/migrations/0037_rpm_primary_repodata_new_location.py

```diff
@@ -0,0 +1,155 @@
+from cStringIO import StringIO
+import sys
+import os
+
+from pulp.server.db import connection
+
+if sys.version < (2, 7):
+    # We need the non-C implementation so we can shove a namespace into its internal data structure,
+    # since in python 2.6, the register_namespace function wasn't present.
+    import xml.etree.ElementTree as ET
+else:
+    import xml.etree.cElementTree as ET
+
+
+RPM_NAMESPACE = 'http://linux.duke.edu/metadata/rpm'
+PULP_PACKAGES_DIR = 'Packages'
+
+# this is required because some of the pre-migration XML tags use the "rpm"
+# namespace, which causes a parse error if that namespace isn't declared.
+FAKE_XML = '<?xml version="1.0" encoding="%(encoding)s"?><faketag ' \
+           'xmlns:rpm="%(namespace)s">%(xml)s</faketag>'
+
+
+def migrate(*args, **kwargs):
+    """Migrate RPMs and SRPMs location to new format "Packages/[a-z]/.*".
+
```

Comment:
I think only one newline is needed and "Migrate ..." should start from a new line.